tracker/src/Tracker/Services/Tracker.hs

```diff
@@ -0,0 +1,53 @@
+module Tracker.Services.Tracker where
+
+import CardanoTx.Models
+
+import Explorer.Service
+import Explorer.Models
+import Explorer.Class
+import Explorer.Types
+
+import System.Logging.Hlog
+import RIO
+import GHC.Natural
+import Control.Retry
+import Control.Monad.Catch
+
+import Tracker.Syntax.Retry
+import Tracker.Caches.Cache
+import Tracker.Models.AppConfig
+
+data TrackerService f = TrackerService
+  { getAllTransactions :: f ([CompletedTx], Int)
+  }
+
+mkTrackerService
+  :: (Monad i, MonadIO f, MonadMask f)
+  => TrackerSettings
+  -> MakeLogging i f
+  -> Cache f
+  -> Explorer f
+  -> i (TrackerService f)
+mkTrackerService settings MakeLogging{..} cache explorer = do
+  logger <- forComponent "trackerService"
+  pure $ TrackerService $ getAllTransactions' explorer cache logger settings  
+
+getAllTransactions' 
+  :: (MonadIO f, MonadMask f) 
+  => Explorer f
+  -> Cache f
+  -> Logging f
+  -> TrackerSettings
+  -> f ([CompletedTx], Int)
+getAllTransactions' Explorer{..} Cache{..} l@Logging{..} TrackerSettings{..} = do
+  _ <- infoM @String "Going to get next tracker iteration."
+  lastIndex <- getLastIndex
+  Items{..} <- execWithRetry l "Retrying getTxn." (getTxs (Paging lastIndex (naturalToInt limit)) Asc)
+  _ <- infoM $ "Got next txn batch: " ++ (show $ length items)
+  let 
+    cardanoTxn = fmap toCardanoTx items
+    newIndex = (length items) + lastIndex + 1
```

Comment:
Why not just tracking index by `Transaction.globalIndex` https://github.com/ergolabs/cardano-explorer-backend/blob/308052ed814d09f8ff793bd2ba64ba73f7324cec/modules/api/src/main/scala/io/ergolabs/cardano/explorer/api/v1/models/Transaction.scala#L22 ?

Reply:
https://github.com/ergolabs/cardano-explorer-backend/blob/308052ed814d09f8ff793bd2ba64ba73f7324cec/modules/core/src/main/scala/io/ergolabs/cardano/explorer/core/db/sql/TransactionsSql.scala#L28
doesn't support gix ordering

Reply:
For example if you call `https://testnet-api.quickblue.io/v1/transactions/?offset=10&limit=1&ordering=asc` the result of **globalIndex** is **4587211**